scripts/Muon/GUI/Common/utilities/load_utils.py

```diff
@@ -211,33 +211,28 @@ def load_workspace_from_filename(filename,
         load_result = _get_algorithm_properties(alg, output_properties)
         load_result["OutputWorkspace"] = [MuonWorkspaceWrapper(ws) for ws in workspace.getNames()]
         run = get_run_from_multi_period_data(workspace)
-        if not psi_data:
-            load_result["DataDeadTimeTable"] = AnalysisDataService.retrieve(load_result["DeadTimeTable"]).getNames()[0]
-            for index, deadtime_table in enumerate(AnalysisDataService.retrieve(load_result["DeadTimeTable"]).getNames()):
-                if index == 0:
-                    load_result["DataDeadTimeTable"] = deadtime_table
-                else:
-                    DeleteWorkspace(Workspace=deadtime_table)
-
-            load_result["FirstGoodData"] = round(load_result["FirstGoodData"] - load_result['TimeZero'], 2)
-            UnGroupWorkspace(load_result["DeadTimeTable"])
-            load_result["DeadTimeTable"] = None
-            UnGroupWorkspace(workspace.name())
-        else:
-            load_result["DataDeadTimeTable"] = None
-            load_result["FirstGoodData"] = round(load_result["FirstGoodData"], 2)
+
+        load_result["DataDeadTimeTable"] = AnalysisDataService.retrieve(load_result["DeadTimeTable"]).getNames()[0]
+        for index, deadtime_table in enumerate(AnalysisDataService.retrieve(load_result["DeadTimeTable"]).getNames()):
+            if index == 0:
+                load_result["DataDeadTimeTable"] = deadtime_table
+            else:
+                DeleteWorkspace(Workspace=deadtime_table)
+
+        load_result["FirstGoodData"] = round(load_result["FirstGoodData"] - load_result['TimeZero'], 2)
+        UnGroupWorkspace(load_result["DeadTimeTable"])
+        load_result["DeadTimeTable"] = None
+        UnGroupWorkspace(workspace.name())
```

Comment:
Why do we insert the deadtimetable into the dict, then remove it after calling delete workspace
Isn't this block the same as?

```
table = deadtime_tables[0]
UnGroupWorkspace(table)
UnGroupWorkspace(workspace.name())
```

Reply:
the above key is different (DataDeadTimeTable instead of DeadTimeTable)